src/lib/process/api.ts

```diff
@@ -9,6 +21,8 @@ export const createProcessTransaction = async (
   options: Polkadot.Options
 ): Promise<Process.Payload> => {
   const sudo = polkadot.keyring.addFromUri(options.USER_URI)
+  
+  if (!isProgramValid(program)) throw new Error('invalid program')
```

Comment:
could be a error like `InvalidProgramError` or something so that when we test we know it's specifically this error. There's an `error.ts` in the repo

Reply:
Oh, sorry did, check utils but not this one.